coordinator/src/services/validation/prescribing-dispensing-tracker.ts

```diff
@@ -0,0 +1,39 @@
+import {getDispenseEnabled, getPrescribeEnabled} from "../../utils/feature-flags"
+import {validationErrors as errors, fhir} from "@models"
+
+const prescribingPermittedScopes = [
+  "urn:nhsd:apim:user-nhs-id:aal3:electronic-prescription-service-api:prescribing"
+]
+
+const dispensingPermittedScopes = [
+  "urn:nhsd:apim:user-nhs-id:aal3:electronic-prescription-service-api:dispensing",
+  "urn:nhsd:apim:app:level3:electronic-prescription-service-api:dispensing"
+]
+
+export function validatePermittedPrescribeMessage(scope: string): Array<fhir.OperationOutcomeIssue> {
+  if (!getPrescribeEnabled()) {
+    return [errors.createDisabledFeatureIssue("Prescribing")]
+  }
+
+  if (!validateScope(scope, prescribingPermittedScopes)) {
```

Comment:
If the scopes from the request include the prescribing _app restricted_ scope, we could return a more specific error message here: "Prescribing is not available using the app-restricted access mode."

Reply:
Made this change.